vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -240,18 +262,21 @@ def invoke_fused_moe_kernel(A: torch.Tensor, B: torch.Tensor, C: torch.Tensor,
         C.stride(2),
         MUL_ROUTED_WEIGHT=mul_routed_weight,
         top_k=top_k,
-        compute_type=tl.bfloat16 if A.dtype == torch.bfloat16 else tl.float16,
+        compute_type=compute_type,
+        use_fp8=use_fp8,
         **config,
     )
 
 
-def get_config_file_name(E: int, N: int) -> str:
+def get_config_file_name(E: int, N: int, dtype: Optional[str]) -> str:
     device_name = torch.cuda.get_device_name().replace(" ", "_")
-    return f"E={E},N={N},device_name={device_name}.json"
+    dtype_selector = "" if not dtype else f",dtype={dtype}"
```

Comment:
It turns out, this actually broke the loading since if we do this, Python will add an additional set of quotes (i.e. `dtype='fp8'` instead of `dtype=fp8` which breaks loading the file. Since this is a file name, it is better not to have those quotes I think, so I just reverted this :)